yarn.lock

```diff
@@ -4647,6 +4655,13 @@ axe-core@^4.4.3:
   resolved "https://registry.yarnpkg.com/axe-core/-/axe-core-4.6.1.tgz#79cccdee3e3ab61a8f42c458d4123a6768e6fbce"
   integrity sha512-lCZN5XRuOnpG4bpMq8v0khrWtUOn+i8lZSb6wHZH56ZfbIEv6XwJV84AAueh9/zi7qPVJ/E4yz6fmsiyOmXR4w==
 
+axios@0.24.0:
```

Comment:
The downside is that we will have to hardcode the URL. Alternatively, it could be exposed by the relay-service. Wdyt?

Reply:
This dependency is only added because we are installing the service directly to get the types. It won't be there when we create a types package (or equivalent).

Reply:
No it is a dependency of the `gelato-relay-sdk` not from our service.
We use the sdk to track the status of `taskId`s

Reply:
Ah, I didn't realise that. We should definitely discuss this.

Reply:
I've removed this dependency